docs_source_files/content/support_packages/working_with_cookies.de.md

```diff
@@ -128,7 +160,28 @@ driver.add_cookie({"name": "foo", "value": "bar"})
 print driver.get_cookie("foo")
   {{< / code-panel >}}
   {{< code-panel language="csharp" >}}
-// Please raise a PR
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+
+namespace GetCookieNamed {
+    class GetCookieNamed {
+        public static void Main(string[] args) {
+            IWebDriver driver = new ChromeDriver();
+            try {
+                // Navigate to Url
+                driver.Navigate().GoToUrl("https://example.com");
+                driver.Manage().Cookies.AddCookie(new Cookie("foo", "bar"));
+
+		        // Get cookie details with named cookie 'foo'
```

Comment:
This looks way too indented to the right, could you please align it?

Reply:
Updated code as suggested